### PR TITLE
Remove build example from docs/experimental-windows.md

### LINKED ISSUE #1032: Update documentation for windows nodes
The [documentation](https://docs.k0sproject.io/v1.21.2+k0s.1/experimental-windows/) for windows claims that you should start with building the windows binary with `make clean k0s.exe`. I believe that needs to be updated because:

- users think this should be done from a windows machine, while we cross compile k0s.exe from linux
- the `make clean` should no longer be needed since 31d3e8b7bd86b6a0fdbbd8e0626836524b30679a
- we provide pre-compiled windows binaries nowdays, so we can probably drop the build part from user documentation

### PR DESCRIPTION
**Pull request by qiuge615**
---
name: Pull Request
about: Create a Pull Request
title: 'Remove Windows build part from user documentation'
labels: 'bug'
assignees: 'geqiu615'

---

**Issue**
Fixes #1032 

**What this PR Includes**
Remove Windows build part from user documentation. Update two typos in the same document.

### CODE REVIEW

File: docs/experimental-windows.md [hunk 1]
```diff
@@ -4,17 +4,13 @@
...
-
-Invoke the `make clean k0s.exe` command to create k0s.exe with staged kubelet.exe and kube-proxy.exe.
+## Run k0s
 
 **Note**: The k0s.exe supervises kubelet.exe and kube-proxy.exe.
 
-During the first run, the calico install script is creaated as `C:\bootstrap.ps1`. This bootstrap script downloads the calico binaries, builds pause container and set ups vSwitch settings.
-
-## Run k0s
+During the first run, the calico install script is created as `C:\bootstrap.ps1`. This bootstrap script downloads the calico binaries, builds pause container and set ups vSwitch settings.
```
Comment (kke) [inline docs/experimental-windows.md:13]: "set ups" should probably be "sets up"

**Review by kke — DISMISSED** [review]
As we already provide precompiled windows binaries, the part about compiling can probably  be left out or moved elsewhere.

LGTM except "set ups" should be "sets up" :)

File: docs/experimental-windows.md (hunk 1 above)
Comment (kke) [reply docs/experimental-windows.md:13]:
```suggestion
During the first run, the calico install script is created as `C:\bootstrap.ps1`. This bootstrap script downloads the calico binaries, builds pause container and sets up vSwitch settings.
```

Comment (qiuge615) [reply docs/experimental-windows.md]: Thank you for your suggestion, I updated this.

**Review by kke — APPROVED** [review]
LGTM.

As we already provide precompiled windows binaries and setting up a dev environment where a simple `make k0s.exe` will work, removing it from documentation is probably a good idea.

**Review by kke — CHANGES_REQUESTED** [review]
You need to do something like `git rebase -i main --signoff` to make the DCO pass.

**Comment by soider** [comment]
closed in favor of #1311

**Pull request closed without merging**